greeneeds/users/views.py

```diff
@@ -1,3 +1,21 @@
-from django.shortcuts import render
+import json
+from datetime         import date
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from core.utils       import login_decorator
+from users.models     import Like
+from projects.models  import Project
+
+class UserInfoView(View):
+    @login_decorator
+    def get(self, request):
+        user = request.user
+
+        result = {
+            'user_id'  : user.id,
+            'nickname' : user.nickname,
+            'email'    : user.email
+        }
+        return JsonResponse({'RESULT': result}, status=200)
```

Comment:
key 값은 주로 스네이크케이스로 사용하는 경우가 많습니다.
```suggestion
        return JsonResponse({'result': result}, status=200)
```